src/main/java/it/gov/pagopa/pu/sil/controller/MassiveExportController.java

```diff
@@ -0,0 +1,99 @@
+package it.gov.pagopa.pu.sil.controller;
+
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.processexecutions.dto.generated.ExportFile;
+import it.gov.pagopa.pu.sil.controller.generated.MassiveExportApi;
+import it.gov.pagopa.pu.sil.dto.generated.ExportFileResponseDTO;
+import it.gov.pagopa.pu.sil.dto.generated.ExportRequestDTO;
+import it.gov.pagopa.pu.sil.mapper.MassiveExportRequestMapper;
+import it.gov.pagopa.pu.sil.security.SecurityUtils;
+import it.gov.pagopa.pu.sil.service.AuthorizationService;
+import it.gov.pagopa.pu.sil.service.exportfile.PaaSILPrenotaExportFlussoIncrementaleConRicevutaService;
+import it.gov.pagopa.pu.sil.service.exportfile.PivotSILPrenotaExportFlussoRiconciliazioneService;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+
+@Slf4j
+@RestController
+public class MassiveExportController implements MassiveExportApi {
+
+  private final PaaSILPrenotaExportFlussoIncrementaleConRicevutaService paaSILPrenotaExportFlussoIncrementaleConRicevutaService;
+  private final PivotSILPrenotaExportFlussoRiconciliazioneService pivotSILPrenotaExportFlussoRiconciliazioneService;
+
+  public MassiveExportController(PaaSILPrenotaExportFlussoIncrementaleConRicevutaService paaSILPrenotaExportFlussoIncrementaleConRicevutaService,
+                                 PivotSILPrenotaExportFlussoRiconciliazioneService pivotSILPrenotaExportFlussoRiconciliazioneService) {
+    this.paaSILPrenotaExportFlussoIncrementaleConRicevutaService = paaSILPrenotaExportFlussoIncrementaleConRicevutaService;
+    this.pivotSILPrenotaExportFlussoRiconciliazioneService = pivotSILPrenotaExportFlussoRiconciliazioneService;
+  }
+
+  @Override
+  public ResponseEntity<ExportFileResponseDTO> massiveExportRequest(String orgFiscalCode, ExportRequestDTO exportRequestDTO) {
```

Comment:
in order to typify the filterFields, I would split this API for each export type...
or I would introduce a polymorphic description of the model, as done on process-executions